image/helpers.go

```diff
@@ -221,15 +221,18 @@ func (tsto *ToolingStore) DownloadSnap(name string, revision snap.Revision, opts
 		targetDir = pwd
 	}
 
-	spec := store.SnapSpec{
+	actions := []*store.SnapAction{{
+		Action:   "download",
 		Name:     name,
-		Channel:  opts.Channel,
 		Revision: revision,
-	}
-	snap, err := sto.SnapInfo(spec, tsto.user)
+		Channel:  opts.Channel,
+	}}
+
+	snaps, err := sto.SnapAction(context.TODO(), nil, actions, tsto.user, nil)
 	if err != nil {
 		return "", nil, fmt.Errorf("cannot find snap %q: %v", name, err)
```

Comment:
should we keep this prefix?  we will get something like  "cannot find snap ...: cannot download: ..."